recipes/taglib/all/conandata.yml

```diff
@@ -1,7 +1,4 @@
 sources:
-  "1.12":
```

Comment:
Please, do not remove the old version. Replacing the only version available can break users.

Reply:
I've just pushed a new commit which allows 1.12 and 1.13 coexisting. However, I wrongly force pushed a synced version of CCI, and the PR autoclosed. Could you please help me reopen this PR?

Reply:
I just pushed a new commit, keeping both 1.12 and 1.13 coexisting.